solidity/test/integration/ConnextVestingWallet.t.sol

```diff
@@ -70,58 +75,52 @@ contract IntegrationConnextVestingWallet is IntegrationBase {
    * It should take into account already withdrawn tokens.
    */
   function test_WithdrawableAmount() public {
-    deal(NEXT_TOKEN_ADDRESS, _connextVestingWalletAddress, 25_000_000 ether);
+    deal(NEXT_TOKEN_ADDRESS, _connextVestingWalletAddress, 15 ether);
 
     assertEq(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 0);
 
-    vm.warp(_unlockStartTime + 364 days);
+    vm.warp(_connextTokenLaunch + YEAR - 1);
     assertEq(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 0);
 
     vm.warp(_firstMilestoneTimestamp);
-    assertEq(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 1_920_000 ether);
+    assertEq(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 1 ether);
 
-    vm.warp(_firstMilestoneTimestamp + 10 days);
-    assertApproxEqAbs(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 2_551_232 ether, MAX_DELTA);
+    vm.warp(_firstMilestoneTimestamp + MONTH);
+    assertApproxEqAbs(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 2 ether, MAX_DELTA);
 
     _connextVestingWallet.release(NEXT_TOKEN_ADDRESS);
     assertEq(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 0 ether);
 
-    // 2,551,232 tokens have been withdrawn
-    vm.warp(_firstMilestoneTimestamp + 100 days);
-    assertApproxEqAbs(
-      _connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 8_232_328 ether - 2_551_232 ether, MAX_DELTA
-    );
+    // 2 ether have been withdrawn
+    vm.warp(_firstMilestoneTimestamp + MONTH * 2);
+    assertApproxEqAbs(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 3 ether - 2 ether, MAX_DELTA);
 
-    vm.warp(_firstMilestoneTimestamp + 365 days);
-    assertApproxEqAbs(
-      _connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 24_960_000 ether - 2_551_232 ether, MAX_DELTA
-    );
+    vm.warp(_firstMilestoneTimestamp + YEAR);
+    assertApproxEqAbs(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 13 ether - 2 ether, MAX_DELTA);
 
-    vm.warp(_firstMilestoneTimestamp + 365 days + 10 days);
-    assertApproxEqAbs(
-      _connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 24_960_000 ether - 2_551_232 ether, MAX_DELTA
-    );
+    vm.warp(_firstMilestoneTimestamp + YEAR + 10 days);
+    assertApproxEqAbs(_connextVestingWallet.releasable(NEXT_TOKEN_ADDRESS), 13 ether - 2 ether, MAX_DELTA);
```

Comment:
why the `- 2 ether` here?
also, plz add a test showing the behaviour of the dust collector:
- can withdraw any other token
- cannot withdraw NEXT until it has released it all
- can withdraw more NEXT (if any) when it has released it all